website/source/api/client.html.md

```diff
@@ -3,15 +3,21 @@ layout: api
 page_title: Client - HTTP API
 sidebar_current: api-client
 description: |-
-  The /client endpoints interact with the local Nomad agent to interact with
-  client members.
+  The /client endpoints are used to access client statistics and inspect
+  allocations running on a particular client.
 ---
 
 # Client HTTP API
 
-The `/client` endpoints are used to interact with the Nomad clients. The API
-endpoints are hosted by the Nomad client and requests have to be made to the
-Client where the particular allocation was placed.
+The `/client` endpoints are used to interact with the Nomad clients. 
+
+Since Nomad 0.8.0, both a client and server can handle client endpoints. For
```

Comment:
Maybe add a note about the security implications and why this feature is useful.